django_project/settings.py

```diff
@@ -37,7 +37,9 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django_extensions',
```

Comment:
I don't see this anywhere being used. Any reasoning why did you add it ?

Reply:
I added it for shell_plus